src/main/java/pl/med/clinic/entity/Procedure.java

```diff
@@ -0,0 +1,32 @@
+package pl.med.clinic.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Table(name = "procedure")
+@NoArgsConstructor
+@Getter
+@Setter
```

Comment:
Settery chyba nie były potrzebne dla hibernate'a ??

Reply:
usunięte, na razie jednak zostawiłam DTO dla procedur na wszelki wypadek, ale wydaje mi się, że jak zaciągam z pliku i robię parsowanie to już nie ma konieczności zabezpieczania się dto, słusznie?